shader/flatten.wgsl

```diff
@@ -198,10 +217,10 @@ fn es_seg_from_params(p0: vec2f, p1: vec2f, params: EulerParams) -> EulerSeg {
     return EulerSeg(p0, p1, params);
 }
 
+// Note: offset provided is scaled so that 1 = chord length
```

Comment:
Consider renaming `offset` to `normalized_offset` or `scaled_offset` or something along those lines (both for `es_seg_eval_with_offset` and `es_params_eval_with_offset`). I think it doesn't hurt to make this obvious, now that the scaling is not done internally.

Reply:
I chose `normalized_offset` and made that consistent throughout.